.github/workflows/release.yml

```diff
@@ -29,17 +29,7 @@ jobs:
             skip-tests: true
           - os-name: Linux-x86_64
             runs-on: ubuntu-22.04
-            target: x86_64-unknown-linux-musl
```

Comment:
i would keep linux arm & aarch64 (with gnu). no reason to remove it its much more common than power pc as an example. unless you removed these because they fail?